app/forms/options_form.rb

```diff
@@ -0,0 +1,14 @@
+require 'reform/form/coercion'
+
+class OptionsForm < Reform::Form
+  feature Coercion
+  property :toc, type: Types::Form::Int,
```

Comment:
If there are only 3 options, would it be better to do something like:

    validates: { inclusion: [ 0, 1, 2] } ?

Still magic but more Ron Weasley than Harry Potter.